standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -3957,10 +3957,10 @@ private void firePreEvent(PreEventContext event) throws MetaException {
     for (MetaStorePreEventListener listener : preListeners) {
       try {
         listener.onEvent(event);
-      } catch (NoSuchObjectException e) {
-        throw new MetaException(e.getMessage());
-      } catch (InvalidOperationException e) {
-        throw new MetaException(e.getMessage());
+      } catch (NoSuchObjectException | InvalidOperationException e) {
+        MetaException me = new MetaException(e.getMessage());
```

Comment:
You do not need to invoke the initCause the HiveMetaException has the following constructor 


 ```
 public HiveMetaException(String message, Throwable cause) {
    super(message, cause);
  }
```

Reply:
`HiveMetaException` seems not be used in `HMSHandler`, please kindly correct me if I misunderstood.

Reply:
Ups you right.